mozilla_django_oidc/auth.py

```diff
@@ -135,6 +135,9 @@ def authenticate(self, **kwargs):
             access_token = token_response.get('access_token')
 
             if import_from_settings('OIDC_STORE_ACCESS_TOKEN', False):
+                session['oidc_access_token'] = access_token
+
+            if import_from_settings('OIDC_STORE_ID_TOKEN', False):
```

Comment:
I don't have deep enough insight (partially blame memory) into the core of this. But this seems like a huge difference.